dbms/src/Storages/KVStore/Utils/AsyncTasks.h

```diff
@@ -15,19 +15,15 @@
 #pragma once
 
 #include <Common/UniThreadPool.h>
-#include <Storages/KVStore/FFI/ProxyFFI.h>
-#include <Storages/KVStore/MultiRaft/Disagg/FastAddPeer.h>
 
 #include <future>
 
 namespace DB
 {
+template <typename Key, typename Func, typename R>
```

Comment:
Intentionally, a `packaged_task` can be created from `Func`, which returns an `R` that is convertible from `result_of(Func)`.